policy/common/methods.go

```diff
@@ -23,10 +23,10 @@ import (
 )
 
 type Methods struct {
-	Comments                    []string `yaml:"comments,omitempty"`
-	CommentPatterns             []Regexp `yaml:"comment_patterns,omitempty"`
-	GithubReview                bool     `yaml:"github_review,omitempty"`
-	GithubReviewCommentPatterns []Regexp `yaml:"github_review_comment_patterns,omitempty"`
+	Comments                    *[]string `yaml:"comments,omitempty"`
```

Comment:
Making this a pointer is not necessary. In Go, slices implicitly work like pointers but we don't usually think about it because a `nil` slice behaves the same as a non-`nil` empty slice. When a user does not specify the `comments` field in configuration, the value will be `nil`.

Reply:
Makes sense. Will revert that back.